go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/go-yaml/yaml v2.1.0+incompatible
 	github.com/gorilla/websocket v1.4.2
 	github.com/mr-tron/base58 v1.1.2
-	github.com/nspcc-dev/dbft v0.0.0-20200531081613-7a39e7b757ac
+	github.com/nspcc-dev/dbft v0.0.0-20200623061135-7d43d409c736
```

Comment:
Isn't it 5a182c20965e08b8857de45bce23652205666be7?

Reply:
It is, that was an old commit.